src/components/FlowList.vue

```diff
@@ -230,20 +229,21 @@
   text-base
 }
 
-.flow-list__divider { @apply
-  mt-4
-}
-
 .flow-list__filters-active { @apply
   text-subdued
   text-sm
   flex
   gap-2
   justify-end
   items-center
+  mt-4
+}
+
+.flow-list__filter-button--filter-open { @apply
+  cursor-pointer
 }
 
-.flow-list__filter-button--filter-active .p-icon { @apply
-  text-link
+.flow-list__filter-button--filter-active:not(.flow-list__filter-button--filter-open) .p-icon { @apply
```

Comment:
high specificity selectors like this are a code smell IMO. Anyway we can avoid this? Seems like a third dynamic class could be used here.